test/parallel/test-crypto.js

```diff
@@ -141,3 +140,42 @@ assert.throws(function() {
 
 // Make sure memory isn't released before being returned
 console.log(crypto.randomBytes(16));
+
+/**
+ * Check if the stream function uses utf8 as a default encoding.
+ **/
+
+function testEncoding(options, assertionHash) {
+  const hash = crypto.createHash('sha256', options);
+  let hashValue = '';
+
+  hash.on('data', (data) => {
+    hashValue += data.toString('hex');
+  });
+
+  hash.on('end', () => {
+    assert.equal(hashValue, assertionHash);
```

Comment:
Can you use `assert.strictEqual` instead of `assert.equal` here, and maybe wrap the `end` listener in `common.mustCall()`?